### DISCUSSION TITLE
Blog with custom image instead of social plugin

### DISCUSSION THREAD

**Discussion by holgerflick** [squidfunk/mkdocs-material#6956] [Q&A]
Is there a way to use a custom image instead of using the social plugin?
Somithing like `Icon:` just with `Image:` for a blog post?

Would also need the dimensions of the image to be provided. I would love an alternative. If this is already possible, please kindly point me to the docs. I went over the blog plugin today and could not find anything. Thank you!

**Comment by squidfunk** [comment]
IIUC, I'm pretty sure this can be resolved via https://github.com/squidfunk/mkdocs-material/issues/5567, more specifically https://github.com/squidfunk/mkdocs-material/issues/5567#issuecomment-1643468553:

``` md
---
social:
  cards_layout: default/only/image
  cards_layout_options:
    background_image: <url>
---

# Page title
```

This can be combined with the [meta plugin](https://squidfunk.github.io/mkdocs-material/plugins/meta/), to set this for _every_ blog post (or a subset in a specific directory). Note that the upside of using a background image is that the social plugin will scale it to fit, so it will always be correctly sized for social media. If you don't want this behavior, you can check the other comments in the issue, as the OP posted a hook that allows to use a custom image.

**Reply by squidfunk** [reply]
Also, with this configuration you can still use the social plugin for the rest of pages (as you specifically asked about changing it for the blog). The plugin can be reconfigured via front matter, as you can see from my previous comment.
